backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Core/Helpers/PolicyHelper.cs

```diff
@@ -141,6 +141,7 @@ public static Rule CreateRuleFromPolicyAndRuleMatch(RequestToDelete search, Xacm
         /// <param name="offeredBy">The party id of the entity offering the delegated the policy</param>
         /// <param name="coveredByUserId">The user id of the entity having received the delegated policy or null if party id</param>
         /// <param name="coveredByPartyId">The party id of the entity having received the delegated policy or null if user id</param>
+        /// <param name="resourceRegistryId">The id of the resource in the resourceregistry.</param>
         /// <returns>policypath matching input data</returns>
         public static string GetAltinnAppDelegationPolicyPath(string org, string app, string offeredBy, int? coveredByUserId, int? coveredByPartyId, string resourceRegistryId)
```

Comment:
Method should be renamed from GetAltinnAppDelegationPolicyPath to GetDelegationPolicyPath as it is no longer altinn app specific